web/api/health/health_cmdapi.c

```diff
@@ -116,7 +116,7 @@ void health_silencers2file(BUFFER *wb) {
  * @return It returns 200 on success and another code otherwise.
  */
 int web_client_api_request_v1_mgmt_health(RRDHOST *host, struct web_client *w, char *url) {
-    int ret = 400;
+    int ret;
```

Comment:
This used to be `int ret = 400;` why did you remove the assignment?

Reply:
I removed it, because it is never returned, as you can see in the whole function, it is set `ret = 403` on lines 134 and 139 and we also set `ret = 200` at line 193, so I believe this function was changed during the time and this `ret = 400` is there due the first versions, because the latest version is overwritten the values before the function ends.